README.md

```diff
@@ -5,13 +5,6 @@
 A modern Wadler/Leijen Prettyprinter
 ====================================
 
-[![](https://img.shields.io/travis/quchen/prettyprinter/master.svg?style=flat-square&label=Master%20build)](https://travis-ci.org/quchen/prettyprinter)
-[![](https://img.shields.io/hackage/v/prettyprinter.svg?style=flat-square&label=Hackage&colorB=0a7bbb)](https://hackage.haskell.org/package/prettyprinter)
```

Comment:
Would you mind if we'd keep the Hackage badge? The Hackage page has useful info such as the Haddocks and compatibility info…

Reply:
Sure, why not. I’ll add it back.